runner/signals_linux.go

```diff
@@ -9,9 +10,10 @@ import (
 
 func signalWatch(cmd *exec.Cmd) {
 	signals := make(chan os.Signal, 1)
-	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
+	signal.Notify(signals, syscall.SIGINT, syscall.SIGQUIT, syscall.SIGTERM)
 	go func() {
 		sig := <-signals
-		syscall.Kill(cmd.Process.Pid, sig.(syscall.Signal))
+		fmt.Println("Received Signal", sig, "sending SIGTERM to child", cmd.Process.Pid)
+		syscall.Kill(cmd.Process.Pid, syscall.SIGTERM)
```

Comment:
Same

Reply:
Well, in that case, it deserves a refactoring: it SIGQUIT is sent, then SIGTERM, it does not behaves as it should If I am not mistaken

Reply:
Yep, signals should be handled in a for loop. You can just propagate the received signal each time.